samples/bluetooth/mtu_update/peripheral/src/peripheral_mtu_update.c

```diff
@@ -56,7 +50,7 @@ void run_peripheral_sample(uint8_t *notify_data, size_t notify_data_size, uint16
 		return;
 	}
 
-	bt_gatt_cb_register(&gatt_callbacks);
+	bt_gatt_cb_register(cb);
```

Comment:
Similarly I would keep this sample's own callback and have the caller register it's own if it needs it.